src/js/views/hallOfFame.js

```diff
@@ -30,11 +30,19 @@ async function updatePlayers(inputs, updateEvents) {
                 min: 0,
                 per: 0,
             };
+            players[i].teamSums = {};
             for (let j = 0; j < players[i].stats.length; j++) {
+                const team = players[i].stats[j].tid;
                 if (players[i].stats[j].gp * players[i].stats[j].min * players[i].stats[j].per > players[i].bestStats.gp * players[i].bestStats.min * players[i].bestStats.per) {
                     players[i].bestStats = players[i].stats[j];
                 }
+                if (players[i].teamSums.hasOwnProperty(team)) {
+                    players[i].teamSums[team] += players[i].stats[j].gp * players[i].stats[j].min * players[i].stats[j].per;
```

Comment:
I forget, what's the difference? EWA is basically GP * min * PER. I would just change these exact formulas to use EWA instead of GP * min * PER. I could be missing something.